packages/wrangler/src/pages.tsx

```diff
@@ -950,7 +1073,7 @@ const createDeployment: CommandModule<
               content: fileContent,
               metadata: {
                 sizeInBytes: filestat.size,
-                hash: hash(content).toString("hex"),
+                hash: hash(content).toString("hex").slice(0, 32),
```

Comment:
This needs to be `16` since you're slicing the hex string and not a byte array

Reply:
Are you sure? I'm getting back a 32 char hash from the API now after those changes we made.

Reply:
And the hashes are matching when I run this build.

Reply:
Let me check again. Hold on.

Reply:
Lol, never mind. You're right. I was confused, my apologies.